save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/basic/organizations/OrganizationTestsMenu.kt

```diff
@@ -123,6 +123,22 @@ private fun organizationTestsMenu() = FC<OrganizationTestsMenuProps> { props ->
             setTestSuitesSourceSnapshotKeys(testSuitesSourceSnapshotKeys.filterNot(it::equals))
         }
     }
+    val refreshTestSuitesSourcesSnapshotKey = useDeferredRequest {
+        val response = get(
```

Comment:
it looks like as a code duplication with line 52-61. Can you please extract common part to a common method and re-use it

Reply:
Fixed